### ISSUE TITLE
Windows Update apparently offering to update (downgrade) to older version 5.10.102.2

### ISSUE THREAD

**Issue by flcdrg** [microsoft/WSL#9435]
### Version

Microsoft Windows [Version 10.0.22621.963]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.15.79.1

### Other Software

WSL version: 1.0.3.0
Kernel version: 5.15.79.1
WSLg version: 1.0.47
MSRDC version: 1.2.3575
Direct3D version: 1.606.4
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22621.963

### Repro Steps

Check Windows Update. See the following:
![image](https://user-images.githubusercontent.com/384747/210661983-ac748217-cb72-449f-8872-b9a869d6ceda.png)

Windows Update is today offering me:
- Windows Subsystem for Linux Update - 5.10.102.2
- Windows Subsystem for Linux WSLg Preview - 1.0.27

### Expected Behavior

These appear to be older versions

### Actual Behavior

Windows Update should not offer older versions

**Comment by benhillis** [comment]
This kernel is for the inbox version of WSL, looks like WU should be updated to not offer the upgrade if the store version of WSL is installed. Thanks for the tip.

**Comment by flcdrg** [comment]
Do you need me to raise a Feedback Hub issue as well, or is this issue enough?

**Comment by benhillis** [comment]
Nope, this is sufficient. Thanks!

**Comment by tebeco** [comment]
@flcdrg / @benhillis 
Is there anything we have to do on our Dev box to avoid that?
Or do we wait for WA team to fix it?
How do we know that WA deployed a fix?

TLDR: how do we track status of that one without pinging you here

context: I have a scenario where WSL2 will absolutly never after
* Disk part => clean all
* win11 install
* bump to dev channel
* all windows update
* `wsl --install `
* reboot
* wsl error

I suspect the `wsl-update` not installed or component getting uninstalled somehow related to this issue

**Comment by tebeco** [comment]
@benhillis is the auto close behavior normal on this issue ?

as in, was that fixed and left open / forgotten

or not fixed and should be open ?

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
Windows Update offered to downgrade WSL to Windows Subsystem for Linux Update - 5.10.102.2 from 
```
wsl --version
WSL-Version: 2.2.2.0
Kernelversion: 5.15.150.1-2
WSLg-Version: 1.0.61
MSRDC-Version: 1.2.5105
Direct3D-Version: 1.611.1-81528511
DXCore-Version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows-Version: 10.0.22631.3374
```

Still same issue:

![Windows Subsystem for Linux Update - 5.10.102.2](https://github.com/microsoft/WSL/assets/46166740/26df0c33-b6b5-4669-8911-0c7b7d5c5244)
